runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ParDoTranslation.java

```diff
@@ -208,8 +208,15 @@ public static ParDoPayload translateParDo(
         new ParDoLike() {
           @Override
           public SdkFunctionSpec translateDoFn(SdkComponents newComponents) {
+            Map<String, PCollectionView<?>> sideInputMapping =
+                parDo.getSideInputs().entrySet().stream()
+                    .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
```

Comment:
I believe that this collector is a noop - it will just create the same map as the input (I think this might be a remnant of my code).

Reply:
Done